composites/Plugin/SnippetPreview/components/HelpTextWrapper.js

```diff
@@ -0,0 +1,139 @@
+import React from "react";
+import PropTypes from "prop-types";
+import styled from "styled-components";
+
+import HelpText from "../../Shared/components/HelpText";
+import colors from "../../../../style-guide/colors.json";
+import { Button } from "../../Shared/components/Button";
+import SvgIcon from "../../Shared/components/SvgIcon";
+import { rgba } from "../../../../style-guide/helpers";
+
+
+const HelpTextContainer = styled.div`
+	max-width: 50em;
+    padding: 1 0 3 1em;
+    font-weight: normal;
+    margin: 0px 20px 10px 25px
+`;
+
+let HelpTextDiv = styled.div`
+	max-width: 30em !important;
```

Comment:
Was this !important really necessary or is it a remnant from experimenting?

Reply:
WHy is it in ems instead of px/%?

Reply:
This was copied from the specifications of the old snippet editor help text, but I have adjusted it to use pixels now instead.